src/test/ui/parser/issue-33418.stderr

```diff
@@ -0,0 +1,64 @@
+error: negative trait bounds are not supported
+  --> $DIR/issue-33418.rs:3:9
+   |
+LL | trait Tr: !SuperA {} //~ ERROR negative trait bounds are not supported
+   |         ^^^^^^^^^ help: remove the trait bound
+
+error: negative trait bounds are not supported
+  --> $DIR/issue-33418.rs:4:19
+   |
+LL | trait Tr2: SuperA + !SuperB {} //~ ERROR negative trait bounds are not supported
+   |                   ^^^^^^^^^ help: remove the trait bound
+
+error: negative trait bounds are not supported
+  --> $DIR/issue-33418.rs:5:10
+   |
+LL | trait Tr3: !SuperA + SuperB {} //~ ERROR negative trait bounds are not supported
+   |          ^^^^^^^^^
+help: replace this `+` with `:`
+   |
+LL | trait Tr3: !SuperA : SuperB {} //~ ERROR negative trait bounds are not supported
+   |                    ^
+help: remove the trait bound
+   |
+LL | trait Tr3 + SuperB {} //~ ERROR negative trait bounds are not supported
+   |         --
```

Comment:
These help messages come out nonsensical, but the code suggestions apply correctly. Is this okay?